### DISCUSSION TITLE
Change Port 80 / 443 to 81/444

### DISCUSSION THREAD

**Discussion by vanGeek** [AdguardTeam/AdGuardHome#3632] [Q&A] [answered]
Hi,

I'm wondering if it is possible to change the http/s listening port to something different? 
AdGuard Home and Nextcloud are both listening on these ports and I want to change one of them. As the certificate process in nextcloud does not work, when a different port is selected, I'm wondering if it disrupts AGH listens on another port? 

Spoken so, how would I change the ports for Adguard? (installed via snap on ubuntu core? 
I found the commands for nextcloud snap.. but would favor to change AGH port. 

Thanks guys!

**Comment by ameshkov** [comment, marked as answer]
You actually can do that via `AdGuardHome.yaml`

**Reply by essoojay** [reply]
how?

**Reply by bogachenkove** [reply]
@ameshkov Andrey, hello. How about adding port editing to the program settings in the dashboard? Is it possible?

**Reply by RobLewis** [reply]
Cannot find `AdGuardHome.yaml` on my Mac.

**Reply by Dynamic5912** [reply]
> Cannot find `AdGuardHome.yaml` on my Mac. 

Go to the installation directory
